draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4177,108 +4448,315 @@ Content-Type: application/cdni; ptype=ci-trigger-collection.v2
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [
+        {
+          "href": "https://www.example.com/hls/71ec0240/index.m3u8",
+          "type": "hls"
+        },
+        {
+          "href": "https://www.example.com/hls/621adfdb/index.m3u8",
+          "type": "hls"
+        }
+      ]
+    }
+  }]
+}
+
+REQUEST:
+
+POST /cit HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
+Content-Type: application/cdni; ptype=ci-trigger.v2
+
+{
+  "action": "preposition",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/b89d49df/index.m3u8",
+        "type": "hls"
+      }]
+     }
+  }],
+  "extensions": [
+    "generic-trigger-extension-type": "execution-policy",
+    "generic-trigger-extension-value": { "priority": 100 }
+   ]   
+}
+RESPONSE:
+
+HTTP/1.1 201 Created
+Date: Sun, 27 Oct 2024 08:48:30 GMT
+Content-Length: 526
+Content-Type: application/cdni; ptype=ci-trigger.v2
+Location: https://dcdn.example/cit/e5483c4a-7c8e-4820-91c8-3c0a9f2edba8
+Server: example-server/0.1
+
+{
+  "ctime": 1730119710,
+  "etime": 1730119770,
+  "mtime": 1730119710,
+  "state": "pending",
+  "action": "preposition",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/b89d49df/index.m3u8",
+        "type": "hls"
+      }]
+     }
+  }],
+  "extensions": [
+    "generic-trigger-extension-type": "execution-policy",
+    "generic-trigger-extension-value": { "priority": 100 }
+   ]   
 }
 ]]></artwork>
-   </figure>
-   </section>
+    </figure>
 
-        <section anchor="trigger-with-extensions" title="Extensions with Error Propagation">
-          <t>
-              In the following example, a CI/T "preposition" command is using two extensions to control the
-              way the trigger is executed. In this example, the receiving dCDN, identified as "AS64500:0",
-              does not support the first extension in the extensions array. dCDN "AS64500:0" further
-              distributes this trigger to another downstream CDN that is identified as "AS64501:0", which
-              does not support the second extension in the extensions array. The error is propagated from
-              "AS64501:0" to "AS64500:0" and the errors.v2 array reflects both errors.
-            </t>
-            <figure>
-              <artwork><![CDATA[
+
+    </section>
+    <section anchor="trigger-with-extensions" title="Extensions with Error Propagation">
+    <t>
+    In the following example, a CI/T "preposition" command is using two extensions to control the
+    "AS64500:0", does not support the first extension in the extensions array. dCDN "AS64500:0" further
```

Comment:
I think you accidentally dropped this line, and now the paragraph doesn't make sense:
"way the trigger is executed. In this example, the receiving dCDN, identified as"

Reply:
re-added from draft 15